content/departments/product-engineering/engineering/enablement/repo-management/internships.md

```diff
@@ -0,0 +1,72 @@
+# Repo Management Internship Info
+
+Welcome to our internship page! We hope you will find useful information about internships on the Repo Management team, our hiring process, and what you can expect us an intern with us!
+
+## Open Intern Roles
+
+We are currently hiring one intern for Summer 2022.
+
+## Hiring Process
+
+Our interview process focuses on identifying somebody that we think will succeed during their internship. Our definition of success has multiple parts:
+
+1.  You owned a real project and delivered real value to our customers
+2.  You enjoyed your work during the summer and didn't feel stressed or anxious
+3.  You enjoyed working with our team and our team enjoyed working with you
+4.  You resonate with Sourcegraph, our values, and our mission
+
+We have multiple steps to learn more about you and for you to learn more about us.
+
+### Recruiter and Hiring Manager Screens
+
+You will meet with our Recruiter and our Engineering Manager to talk about your background and why you're a good fit for this role and our team. We want to learn about your interests, strengths, and why you're excited about Sourcegraph and Repo Management. Our goal is to determine if you would enjoy this internship and find success as defined above.
+
+### Pairing Interviews
+
+Once you clear the Recruiter and Hiring Manager screens, we will ask you to collaborate with our team on two, 1 hour pairing interviews. We think these interviews are the best way to determine if you will enjoy working with our team, if we will enjoy working with you, and if you have the technical foundations to succeed during the internship.
+
+##### How We'll Work
+
+1. We will pair with you on a scheduled 1 hour call.
+2. We expect you to share your screen during the session.
+3. We also expect that there will be communication from both sides of the call.
+4. You can pick any programming language/library that you are familiar with _(Go is preferred)_
+5. You can use online documentation
+6. It is OK to ask questions or guidance as you would a fellow developer you work with
+
+##### What We're Looking For
+
+1. Someone we want to work with
+2. Code that is working correctly
+3. Code that is easy to understand, maintain, extend in the future
+4. Someone that can explain the code and their thought process to fellow developers
+5. Someone that can communicate well and positively reacts to feedback
+
+We have two different interview prompts, detailed below with a few specifics revealed during the interview itself.
```

Comment:
Would we be sending the instructions 2 hours before the interview to let the candidate get acquainted with the requirement of the task(s)? We've done this for full time roles AFAIK.

Reply:
I left a few comments on our Slack channel last week, but my preference is that we don't send the task out 2 hours before. It can create an unfair advantage for those who have time available before the interview. But people with other obligations (family, work, school, etc) wouldn't have that benefit.

I'd rather provide the outline of the problem here and then review the specifics during the interview. Which means we need to handle as much setup as we can before the call.

Reply:
I must have missed it on Slack. :+1: